main/tests/server/src/com/google/refine/exporters/CsvExporterTests.java

```diff
@@ -224,6 +224,66 @@ public void exportCsvWithEmptyCells() {
                 ",row2cell1,row2cell2\n");
     }
 
+    /*
+    check if CSV exporter can handle a semi-colon as a custom separator
+     */
+    @Test
+    public void exportCsvWithCustomSeparatorSemicolon() {
+        CreateGrid(2, 2);
+        when(options.getProperty("options")).thenReturn("{\"separator\":\";\"}");
+
+        try {
+            SUT.export(project, options, engine, writer);
+        } catch (IOException e) {
+            Assert.fail("IOException should not have occurred", e);
+        }
+
+        assertEqualsSystemLineEnding(writer.toString(), "column0;column1\n" +
+                "row0cell0;row0cell1\n" +
+                "row1cell0;row1cell1\n");
+    }
+
+    /*
+    check output does not contain any quotes around the values, as expected when quoting is turned off
+     */
+    @Test
+    public void exportCsvWithoutQuotingFieldsExplicitly() {
```

Comment:
There seems to be other test cases which also have `quoteAll` turned off (although not explicitly), so I am not sure why we would need this test case. Could you elaborate on what motivated you to add it?